### ISSUE TITLE
Problem: KGW is not in CI

### ISSUE THREAD

**Issue by KwilLuke** [kwilteam/kwil-db#625]
We've run into an issue a few times where our partners have hit bugs in KGW, usually because of a versioning issue (e.g. some change in kwil-db necessitates a KGW change), and we have not caught those issues because KGW is not part of the kwil-db testing.

To address this, we should add the KGW tests to the kwil-db CI. Because we now have multiple partners using KGW in production, we need to ensure that KGW always works with each kwil-db release (we can assume that partners will not be able to use kwil-db without a functioning KGW). IMO, the cleanest way to ensure this is to add kgw to the kwil-db CI.

Is anyone in a good position to pick this up sometime this week?

**Comment by Yaiba** [comment]
#626 and #629 are target this issue. Not perfect but a way to do it.

**Comment by Yaiba** [comment]
KGW test will be triggered once a PR is merged. Consider this is done.
